packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -695,7 +699,60 @@ describe('plugin-meetings', () => {
           assert.isRejected(meeting.setRemoteQualityLevel('LOW'));
         });
       });
+
+      describe('#setMeetingQuality', () => {
+        let mediaDirection;
+
+        beforeEach(() => {
+          mediaDirection = {
+            receiveAudio: true, receiveVideo: true, receiveShare: false, sendVideo: true
+          };
+          meeting.setRemoteQualityLevel = sinon.stub().returns(Promise.resolve());
+          meeting.setLocalVideoQuality = sinon.stub().returns(Promise.resolve());
+          meeting.mediaProperties.mediaDirection = mediaDirection;
+        });
+
+        it('should have #setMeetingQuality', () => {
```

Comment:
```suggestion
        it('should exist as a meeting object method', () => {
```

Reply:
This is just to keep consistent with the rest of the tests in this suite.